src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -53,15 +53,15 @@ const AnonymousChannelListScreen = ({route}) => {
     <>
       <StatusBar translucent={false} />
       <View style={{height: 52}}>
-        <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+        <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />
       </View>
 
       <FlatList
         data={anonChannels}
         keyExtractor={(item) => item.id}
         ListHeaderComponent={
           <ChannelListHeaderItem
-            name="Anonymous"
+            name="Incognito"
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Search` component now receives an additional prop `isAnon` set to `true`, and the `onPress` function now includes an object with a `from` property set to `ANONYMOUS`. This seems like a good addition if the `Search` component and `goToContactScreen` function are designed to handle these changes. Also, the name "Anonymous" has been changed to "Incognito". Ensure that this change is reflected everywhere in the UI for consistency.
```diff
- <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
+ <Search route={route} isAnon={true} onPress={() => goToContactScreen({from: ANONYMOUS})} />

- name="Anonymous"
+ name="Incognito"
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->